enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -143,7 +143,7 @@ The kube-apiserver provides three CLI flags to dictate log rotation and retentio
 
 * `--audit-log-maxage` defines the maximum number of days to retain old audit log files.  MicroShift defaults to 0, disabling the age limit.
 * `--audit-log-maxbackup` defines the maximum number of audit log files to retain. MicroShift defaults to 10 files.
-* `--audit-log-maxsize` defines the maximum size in megabytes the live log file before it is rotated.  Defaults to 200Mb.
+* `--audit-log-maxsize` defines the size of rotated log files in megabytes. Kube-apiserver overrides a 0 value to 100MB.  If unset, MicroShift default is 200MB.
```

Comment:
You removed mention of "live log file" but it still applies, right? Size of all the log files (live and any amount of rotated) will be limited by this setting, right?

Reply:
Right

Reply:
What do you think about including this information? 
```suggestion
* `--audit-log-maxsize` defines the size of rotated and live log files in megabytes. Kube-apiserver overrides a 0 value to 100MB.  If unset, MicroShift default is 200MB.
```

Reply:
Should we forbid the `0` value? MicroShift already defaults to `200` and `0` value translates to a different thing which is "internal". If `0` meant unbound it would make more sense to me than `100`?

Reply:
`0` is still a valid number as far as the kube-apiserver is concerned, so I think so long as documentation makes it clear, it shouldn't block microshift from starting. at most I'd suggest a warning in logs